File: GarageWizardClient/src/app/models/userLogin.ts

export class UserLogin
{
    Email! : string;
    Password! : string;
    RememberMe : boolean = false;

    constructor(email: string, password: string, rememberMe: boolean)
    {
        this.Email = email;
        this.Password = password;
        this.RememberMe = rememberMe;
    }
}

File: GarageWizardAPI/GarageCoreMVC/Models/UserLogin.cs
using Microsoft.AspNetCore.Identity;
using System.ComponentModel.DataAnnotations;

namespace GarageCoreMVC.Models
{
    public class UserLogin
    {
        [Required(ErrorMessage = "Email is required")]
        [EmailAddress]
        public required string Email { get; set; }

        [Required(ErrorMessage = "Password is required")]
        [DataType(DataType.Password)]
        public required string Password { get; set; }

        [Display(Name = "Remember me?")]
        public bool RememberMe { get; set; }

    }
}
